app/presenters/FacultyPresenter.php

```diff
@@ -0,0 +1,99 @@
+<?php
+/**
+ * Created by PhpStorm.
+ * User: Michal Struna
+ * Date: 24/11/2018
+ * Time: 16:30
+ */
+
+namespace App\Presenters;
+
+use App\Model\FacultyModel;
+use Nette\Database\DriverException;
+use Nette\Application\UI\Form;
+use Nette\Utils\ArrayHash;
+
+class FacultyPresenter extends BasePresenter {
+
+    private $facultyModel;
+
+    public function __construct(FacultyModel $facultyModel) {
+        $this->facultyModel = $facultyModel;
+    }
+
+    /**
+     * Create edit faculty form.
+     * @return Form Edit faculty form
+     */
+    protected function createComponentEditFacultyForm(): Form {
```

Comment:
edit form? 🤔 to je spolecny na oboje, ne?

Reply:
Já nevím, jak to pojmenovat. Prostě edituješ už stávající, nebo edituješ novej. :D

Reply:
Okay 😅